e2e/app/sidebar/rstudio-configuration-panel.ts

```diff
@@ -0,0 +1,31 @@
+import { Page } from 'puppeteer';
+import { SideBarLink } from 'app/text-labels';
+import BaseEnvironmentPanel from './base-environment-panel';
+import Button from '../element/button';
+import AppsPanel from './apps-panel';
+
+const defaultXpath = '//*[@id="rstudio-configuration-panel"]';
+
+export default class RStudioConfigurationPanel extends BaseEnvironmentPanel {
+  constructor(page: Page) {
+    super(page, defaultXpath, SideBarLink.RStudioConfiguration);
+  }
+
+  async startRStudioGkeApp(): Promise<void> {
```

Comment:
I'm not familiar with returning `Promise<void>`. What is this used for?

Reply:
All asynchronous functions return Promises, and [void](https://www.typescriptlang.org/docs/handbook/2/functions.html#void) is the return type of functions that return no data. So a `Promise<void>` is an asynchronous function that returns no data.